book/src/intro.md

```diff
@@ -8,15 +8,36 @@ minimal test case to reproduce the problem. Unlike QuickCheck, generation
 and shrinking is defined on a per-value basis instead of per-type, which
 makes it more flexible and simplifies composition.
 
-## Status of this crate
+## Ecosystem
 
-The crate is fairly close to being feature-complete and has not seen
-substantial architectural changes in quite some time. At this point, it mainly
-sees passive maintenance.
+Proptest has evolved to be more than a single library. Currently the following
+crates exist:
+- `proptest` - the core testing library containing traits, implementations of those
+  traits for `core`, `alloc`, `std` and some popular 3rd party crates, a test runner
+  and some sugar to make working with the test runner and traits easier
+- `proptest-derive` - a derive macro for implementing `Arbitrary` when you don't
+  have unique requirements for generating data
+- `proptest-macro` - procedural macros that make writing proptests easier
+- `proptest-state-machine` - a lightweight framework built on top of `proptest`
+  for testing systems against a reference implementation
```

Comment:
adding some periods and removing use of "you", just a suggestion, feel free to ignore if you disagree :)
```suggestion
- `proptest` - the core testing library containing traits, implementations of those
  traits for `core`, `alloc`, `std` and some popular 3rd party crates, a test runner
  and some sugar to make working with the test runner and traits easier.
- `proptest-derive` - a derive macro for implementing `Arbitrary` when the applicable type does not
  have unique requirements for generating data.
- `proptest-macro` - procedural macros that make writing proptests easier.
- `proptest-state-machine` - a lightweight framework built on top of `proptest`
  for testing systems against a reference implementation.
```